SETS/README.md

```diff
@@ -1,49 +1,61 @@
 # Parameters and parameter values read from input text file:
 
 ##  Simulation start/restart from breakpoint files
-- `irestart`: 0 if simulation starts from beginning, 1 if simulation restarts from a step. Default: 00000
-- `istep`: last step written, indicates step to restart from if irestart is 1. Default: 0
+| Parameter  | Description                                                                 | Default value | Allowed values |
```

Comment:
I did not notice that new file, but it looks great. Would be also nice to add a link to the main Readme

Reply:
if you or @halterw  knows how to do so, it will be welcome!